src/Linkify/Linkify.js

```diff
@@ -1,19 +1,49 @@
 import React from 'react';
-import styled from 'styled-components';
 
-const Container = styled.span`
-  a {
-    color: inherit;
-    text-decoration: underline;
-  }
-`;
+const URL_REGEX = /(\b(https?|ftp|file):\/\/[-A-Z0-9+&@#/%?=~_|!:,.;]*[-A-Z0-9+&@#/%=~_|])/gi;
```

Comment:
Do we want to add some rules around domain extensions so that URL's like `duckduckgo.com` in the playground below will also be converted into links?

Reply:
In the future, probably yes. We can add rules to our doctors for now :)

Reply:
@kylealwyn cool. LGTM 🚢